apps/ui/src/components/Ui/Textarea.vue

```diff
@@ -37,7 +37,7 @@ watch(model, () => {
   <UiWrapperInput :definition="definition" :error="error" :dirty="dirty">
     <textarea
       v-model="inputValue"
-      class="s-input"
+      class="s-input !py-0 !border-t-[30px] !border-b-[7px] border-transparent"
```

Comment:
```suggestion
      class="s-input !py-0 !border-t-[30px] border-transparent"
```

I think it looks better without padding at bottom, but maybe we should ask Fab. With current styles we get some text cut off.

<img width="537" alt="image" src="https://github.com/snapshot-labs/sx-monorepo/assets/1968722/6edcbe21-c953-40da-b889-0ffb49b648f2">

Reply:
@bonustrack

Reply:
Agree, also would be better to have a gradient on top, similar than the sidebar
<img width="181" alt="image" src="https://github.com/snapshot-labs/sx-monorepo/assets/16245250/cdce728f-62e5-4aaf-a815-f8f1ea2a772d">